kqueen_ui/asset/dynamic/scss/main.scss

```diff
@@ -115,31 +115,43 @@ a, input[type="checkbox"] {
   pointer-events: none;
 }
 
+.provisioner-pie {
+  line-height: 1.1;
```

Comment:
And why do you need to change it here?

Reply:
It is a code movement, I extract Login section